src/test/java/seedu/address/model/ModelManagerTest.java

```diff
@@ -102,31 +105,55 @@ public void equals() {
 
         // same object -> returns true
-        assertTrue(modelManager.equals(modelManager));
+        assertEquals(modelManager, modelManager);
 
         // null -> returns false
-        assertFalse(modelManager.equals(null));
+        assertNotEquals(null, modelManager);
 
         // different types -> returns false
-        assertFalse(modelManager.equals(5));
+        assertNotEquals(5, modelManager);
 
         // different eduMate -> returns false
-        assertFalse(modelManager.equals(new ModelManager(differentEduMate, userPrefs)));
+        assertNotEquals(modelManager, new ModelManager(differentEduMate, userPrefs));
 
         // different filteredList -> returns false
-        String[] keywords = ALBERT.getName().fullName.split("\\s+");
-        modelManager.updateFilteredPersonList(new NameContainsKeywordsPredicate(Arrays.asList(keywords)));
-        assertFalse(modelManager.equals(new ModelManager(eduMate, userPrefs)));
+        create_equals_filteredList(
+                CliSyntax.PREFIX_NAME, ALBERT.getName().fullName.split("\\s+"), eduMate, userPrefs);
+        create_equals_filteredList(
+                CliSyntax.PREFIX_EMAIL, ALBERT.getEmail().value.split("\\s+"), eduMate, userPrefs);
+        create_equals_filteredList(
+                CliSyntax.PREFIX_PHONE, ALBERT.getPhone().value.split("\\s+"), eduMate, userPrefs);
+        create_equals_filteredList(
+                CliSyntax.PREFIX_ADDRESS, ALBERT.getAddress().value.split("\\s+"), eduMate, userPrefs);
+        create_equals_filteredList(
+                CliSyntax.PREFIX_TELEGRAM_HANDLE, ALBERT.getTelegramHandle().telegramHandle.split("\\s+"),
+                eduMate, userPrefs);
+
+        create_equals_filteredList(
+                CliSyntax.PREFIX_MODULE_TAG,
+                ALBERT.getModuleTags().toString().replaceAll("[\\[\\], ]", "").split(" "),
+                eduMate, userPrefs);
+
+        create_equals_filteredList(
+                CliSyntax.PREFIX_GROUP_TAG,
+                ALBERT.getGroupTags().toString().replaceAll("[\\[\\], ]", "").split(" "),
+                eduMate, userPrefs);
 
         // resets modelManager to initial state for upcoming tests
         modelManager.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
 
         // different userPrefs -> returns false
         UserPrefs differentUserPrefs = new UserPrefs();
         differentUserPrefs.setEduMateFilePath(Paths.get("differentFilePath"));
-        assertFalse(modelManager.equals(new ModelManager(eduMate, differentUserPrefs)));
+        assertNotEquals(modelManager, new ModelManager(eduMate, differentUserPrefs));
+    }
+
+    public void create_equals_filteredList(Prefix prefix, String[] keywords, EduMate eduMate, UserPrefs userPrefs) {
+        modelManager.updateFilteredPersonList(
+                new ContainsKeywordsPredicate(Arrays.asList(keywords), prefix));
+        assertNotEquals(modelManager, new ModelManager(eduMate, userPrefs));
```

Comment:
Non-test cases should not be in snake_case.

Reply:
Changed to camel case.